src/components/create-task-modal/create-task-modal.component.coffee

```diff
@@ -8,21 +8,26 @@ angular.module('mnoUiElements').component('mnoCreateTaskModal', {
   controller: ()->
     ctrl = this
 
+    ctrl.loading = false
     ctrl.$onInit = ->
       ctrl.task = {}
       ctrl.isDraft = !_.isEmpty(ctrl.resolve.draftTask)
-      ctrl.recipients = _.map(ctrl.resolve.recipients, (r)->
-        { id: r.id, name: r.user.name }
-      )
+      ctrl.datepicker = {
+        options: { format: 'dd MMMM yyyy' }
+        opened: false
+      }
+      ctrl.recipients = _.map(ctrl.resolve.recipients, (orgaRel) -> {id: orgaRel.id, name: ctrl.resolve.recipientFormater(orgaRel)})
       if ctrl.isDraft
         draft = ctrl.resolve.draftTask
         recip = draft.task_recipients[0]
-        ctrl.selectedRecipient = { id: recip.orga_relation_id, name: recip.user.name }
+        ctrl.selectedRecipient = {id: recip.orga_relation_id, name: ctrl.resolve.recipientFormater(recip)}
         ctrl.task = _.pick(draft, ['id', 'title', 'message'])
-        ctrl.task.due_date = moment(draft.due_date).toDate() if draft.due_date?
+        ctrl.taskDueDate = moment.utc(draft.due_date).toDate() if draft.due_date?
 
     ctrl.ok = (status = 'sent')->
+      ctrl.loading = true
       angular.merge(ctrl.task, status: status, orga_relation_id: ctrl.selectedRecipient.id)
+      ctrl.task.due_date = moment.utc(ctrl.taskDueDate).toISOString()
```

Comment:
No, `ctrl.taskDueDate` is the datepicker model which does not like ISO strings :)